src/protocol/DocumentServer.js

```diff
@@ -222,6 +223,33 @@ define('polymer-designer/protocol/DocumentServer', [
       });
     }
 
+    backspaceText(request) {
+      let text = request.message.text;
+      let node = document.createTextNode(text);
```

Comment:
@justinfagnani. I did some more testing. If I rename to `node` to `what` it does not work (cursor does not move, just like when removing 'node'.  Also found with `node` it works when you backspace only, but if you add some text then backspace , backspacing stops working.

I will play with it some more. Not having much luck so far. Probably will not have a chance till tomorrow. Converting Road to Polymer to 0.9 currently.